spec/helpers/review_mapping_helper_spec.rb

```diff
@@ -668,4 +668,50 @@
     end
   end
 
+  describe 'feedback_response_map_record' do
+    before(:each) do
+      review_grade = create(:review_grade)
+      @reviewer = create(:participant, review_grade: review_grade)
```

Comment:
Is review grade necessary (Just curious)?

Reply:
Probably not. I just copied and pasted wholesale from another test ('get_color' I think). I'll push a patch to remove it in a minute.